electrum/gui/qt/main_window.py

```diff
@@ -3118,3 +3122,14 @@ def save_transaction_into_wallet(self, tx: Transaction):
                      "to see it, you need to broadcast it."))
             win.msg_box(QPixmap(icon_path("offline_tx.png")), None, _('Success'), msg)
             return True
+
+    def show_cert_mismatch_error(self):
+        if self.showing_cert_mismatch_error:
+            return
+        self.showing_cert_mismatch_error = True
+        self.show_critical(title=_("Certificate mismatch"),
+                           msg="\n\n" +
```

Comment:
Why the leading newlines? It looks a bit weird on both Ubuntu and Windows.
I see from your picture that on macOS it does not show the title for the popup -- weird. In any case I think we should rm these newlines, at least for uniformity with other popups.

Reply:
Ah, I was wondering why that didn't show!

The leading newlines were original because I had "Certificate mismatch" at the beginning of `msg` because it wasn't showing in `title`. When I removed the duplicate "Certificate mismatch" I must've forgotten to remove the leading newlines along with it.